### DISCUSSION TITLE
自己紹介スレ

### DISCUSSION THREAD

**Discussion by Haresaku** [pixiv/vroid-sdk-developers#3] [General]
初めてここにきた人は、自己紹介をして人間強度を下げていきましょう😌

**Comment by Haresaku** [comment]
VRoid Hub / SDKの開発者のHaresakuです。よろしくお願いいたします

**Comment by xys20071111** [comment]
A noob of Unity and VRoid SDK,hoping learn something from here.

**Comment by FujiSunflower** [comment]
2対2のロボット戦闘ゲームの[Roduet](https://www.youtube.com/watch?v=Jq6TEBPAmWw)を作ったりしています。[pixivFANBOX](https://fujisunflower.fanbox.cc/)での支援も歓迎しております。  
開発者コミュニティサイトが移転になったことをほっとしております。

**Comment by nyamadan** [comment]
VRoid Hub / SDKの開発者のnyamadanです。よろしくお願いいたします。
@Haresaku とはVRoid SDKについて日常的にディスカッションできる立場ですが、このディスカッションをオープンにしたほうがいいと思いましたので、こちらに投稿していこうと思います。

**Comment by vdyakuin** [comment]
Unityでゲーム等を作ったりバ美肉おじさんをしたりしています。よろしくお願いいたします。
成人向け対戦ゲームを作ろうとしています。

**Comment by ytomi-pixiv** [comment]
VRoid Hub/SDKのプロダクトマネージャーのYtomiです。
よろしくお願いします！

**Comment by Yas-Utsubo** [comment]
UnityもVRoidも初めてです。頑張って開発しようとしております。皆様、よろしくお願いいたします。
